presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -265,6 +265,28 @@ public void testDate()
         }
     }
 
+    @Test
+    public void testEnum()
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(postgreSqlServer.getJdbcUrl());
+        jdbcSqlExecutor.execute("CREATE TYPE enum_t as enum ('a','b','c')");
```

Comment:
nit: upper case "AS ENUM" as in Postgres docs https://www.postgresql.org/docs/11/datatype-enum.html